src/cnglob.h

```diff
@@ -14,6 +14,17 @@
 // CSE_DLL	build "silent" CSE DLL, screen output returned via callback
 // else CSE_CONSOLE  build console app, screen output to cmd window
 
+// #defines for different operating systems
+// _WIN32			Defined for both 32-bit and 64-bit environments 1
+// _WIN64			Defined for 64-bit environments 1
```

Comment:
Maybe add "windows" to the commends for windows macros, if only for consistency. Also there's a dangling character at the ends of the lines?